trading/analysis/technical/weekly_indicators/lib/ema.mli

```diff
@@ -0,0 +1,7 @@
+open Core
+open Types
+
+val calculate_30_week_ema : Daily_price.t list -> (Date.t * float) list
```

Comment:
I am thinking there should be two functions, one computing it from a daily price series, the other computing it from a weekly prices series. So the former = daily_to_weekly + the latter